src/pydocstyle/cli.py

```diff
@@ -53,7 +53,7 @@ def run_pydocstyle(use_pep257=False):
 
     count = 0
     for error in errors:
-        sys.stderr.write('%s\n' % error)
+        sys.stdout.write('%s\n' % error)
```

Comment:
This is from another PR. Please make sure that only relevant changes are included (I'll do the review after you do this).